storage/cloud-client/acl_test.py

```diff
@@ -39,37 +36,29 @@
 )
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def test_bucket():
-    """Yields a bucket that resets its acl after the test completes."""
-    bucket = storage.Client().bucket(BUCKET)
-    acl = google.cloud.storage.acl.BucketACL(bucket)
-    object_default_acl = google.cloud.storage.acl.DefaultObjectACL(bucket)
-    acl.reload()
-    object_default_acl.reload()
-    time.sleep(1)  # bucket ops rate limited 1 update per second
+    """Yields a bucket that is deleted after the test completes."""
+    bucket = None
+    while bucket is None or bucket.exists():
```

Comment:
Would existence imply that a test somehow somewhere didn't clean up after itself?

Reply:
This is mostly to avoid UUID collisions. I think python forms them with timestamps so, in theory, the parallel cases could try to make the same buckets